### ISSUE TITLE
Is mocha-webpack still maintained?

### ISSUE THREAD

**Issue by larixer** [zinserjan/mocha-webpack#289]
@zinserjan Thank you for the great work on a great tool!

Let me kindly ask a straight question, is `mocha-webpack` still maintained? 

It looks strange that Pull Requests from contributors starting from end of March do not get any reaction from you and that many greenkeeper PR's and issues hang around, no new commits being pushed... It looks like as if project was abandoned from some reason.

Could you shed some light for your users?

Many thanks!

**Comment by alecf** [comment]
It looks like @zinserjan hasn't been active on github (or anywhere else?) since early may 2018... I hope they are ok!

we've been using the 2.0.0-beta.0 version and it works *reasonably* well, but it would be great to get a release out on NPM. At what point does someone request to take this over on NPM?

**Comment by larixer** [comment]
@alecf Thanks for reply. I hope @zinserjan are well too!

Yes, `2.0.0-beta.0` works for now for us too. However things change in JS very fast, Webpack 2, 3, 4 had breaking changes and were released in a relatively short time frame. So, if there will be no reply from @zinserjan  for a while and the last released version of `mocha-webpack` becomes outdated, I think maybe we should fork and publish `mocha-webpack2` on npm?

**Comment by darrynten** [comment]
I'd vote to call it mocha-webpack-4

**Comment by notyoyoma** [comment]
Hey guys, Webpack 4 has been out since March, and I need to get this project working again. Has anyone forked this, and updated for Webpack 4?

@zinserjan If you still get these notifications, would you consider moving mocha-webpack over to the webpack-contrib repo? (similar to [karma-webpack](https://github.com/webpack-contrib/karma-webpack))

**Comment by larixer** [comment]
@notyoyoma The latest version `2.0.0-beta.0` of `mocha-webpack` works for Webpack 4, no need to fork to get working version.

**Comment by notyoyoma** [comment]
@vlasenko I'm already using `2.0.0-beta.0` but debugging [is broken](https://github.com/zinserjan/mocha-webpack/issues/266) probably due to Webpack 4.

**Comment by larixer** [comment]
@notyoyoma It doesn't mean though that the project is not working for Webpack 4. And it doesn't mean that debugging has ever worked the way you want it to.

**Comment by notyoyoma** [comment]
@vlasenko I will look again, and try to get the debugger working as described [here](https://github.com/zinserjan/mocha-webpack/blob/master/docs/guides/ide-integration.md), but I have already spent ~20hrs on this problem over several weeks, and am fairly certain the problem lies with mocha-webpack.

Regardless, there still has been no response or activity from zinserjan in the last month. Does anyone know if he has a twitter or other social media presence? I've looked but haven't found anything. If he doesn't return, we may need to fork the project to continue development.

**Comment by ViggoV** [comment]
I'd be thrilled if someone with the nescessary expertise would resurrect this project. I've been experimenting with `2.0.0-beta.0` with some success but also some hacking and tweaking that might undermine the integrity of the tests. Also it's a question of time before Webpack bumps again.

**Comment by darrynten** [comment]
A dependency in `2.0.0-beta.0` has known vulnerabilities.

DepShield reports that this application's usage of lodash.debounce:4.0.8 results in the following vulnerability(s):

(CVSS 7.4) CWE-471: Modification of Assumed-Immutable Data (MAID)

`lodash.debounce:4.0.8` is a transitive dependency introduced by the following direct dependency:

```
• mocha-webpack:2.0.0-beta.0
        └─ chokidar:2.0.4
              └─ lodash.debounce:4.0.8
```

See #297

**Comment by larixer** [comment]
@darrynten Yeah, looks like we are one step closer to the need of forking this repo and maintaining the fork....

**Comment by larixer** [comment]
Hey everyone! I have created the fork of this repository named `mochapack`:
https://github.com/sysgears/mochapack

I and my team is going to maintain it, until we see any activity from original author. Nothing too heavy - we will do security updates mostly and accept pull requests.

The first version is published into npm `mochapack@1.0.0`, it has the source code of latest released `mocha-webpack` version, renamings and update of dev dependency `node-sass` to enable builds of `mochapack` on Node 10.x versions.

**Comment by EmperorArthur** [comment]
Thank you so much!  I can confirm `mochapack` is a drop in replacement for `mocha-webpack`!\*  @larixer you may want to try to contact GitHub or NPM about this being abandoned, so a note can be added to the front page pointing to the replacement.  It took me two solid days of searching for a fix before discovering this issue and `mochapack`.  At the least, you may consider watching this repo and responding to all issues with your new fork.

* With the exception of this issue:  https://github.com/JeffreyWay/laravel-mix/issues/2638
